test/testlib.py

```diff
@@ -586,6 +590,10 @@ def login_and_go(self, path=None, user=None, host=None):
         "(audit: )?type=1403 audit.*",
         "(audit: )?type=1404 audit.*",
 
+        # Happens on rhel-atomic
+        "type=1400 .*granted.*comm=\"tuned\".*",
+        "type=1400 .*denied.*comm=\"iptables\".*name=\"xtables.lock\".*",
```

Comment:
Interestingly, this is a race during boot.  If kube is started early enough after boot, the messages appear.  If we wait 5 seconds before starting kube, they don't appear.